qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -162,11 +147,11 @@ def _check_configuration(self):
                 str(self._num_electrons),
             )
 
-        if isinstance(self._num_molecular_orbitals, (int, np.integer)):
-            if self._num_molecular_orbitals < 0:
+        if isinstance(self._num_spatial_orbitals, (int, np.integer)):
+            if self._num_spatial_orbitals < 0:
                 raise QiskitNatureError(
                     "The number of active orbitals cannot be negative, not:",
-                    str(self._num_molecular_orbitals),
+                    str(self._num_spatial_orbitals),
```

Comment:
The error message below has the wrong field in it
```
                "The number of active orbitals must be an int, not:",
                str(self._num_electrons),
```
Its referring to num_electrons and should be num_spatial_orbitals. I suppose another way could be to cast it to an int, instead of a type check, which would raise an exception if it could not be cast - but maybe leads to other more subtle problems if it were a float.